arch/AArch64/AArch64BaseInfo.c

```diff
@@ -17,7 +17,8 @@
 #ifdef CAPSTONE_HAS_ARM64
 
 #if defined (WIN32) || defined (WIN64) || defined (_WIN32) || defined (_WIN64)
-#pragma warning(disable:4996)
+#pragma warning(disable:4996)			// disable MSVC's warning on strcpy()
+#pragma warning(disable:28719)		// disable MSVC's warning on strcpy()
```

Comment:
looks like a copy & paste: is this 28719  about strncpy(), not strcpy()?
there are few other places about 28719, with the same comment.